### PR TITLE
Refactor: refine cli commands

### PR DESCRIPTION
**Pull request by wonderflow**
### Description of your changes

1. refine cli help message
2. align kubectl-vela with vela, use the same function
3. use getNamespaceAndEnv for all

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by barnettZQG — APPROVED** [review]

File: references/cli/cluster.go [hunk 1]
```diff
@@ -71,10 +71,10 @@ const (
 func ClusterCommandGroup(c common.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "cluster",
-		Short: "Manage Clusters",
-		Long:  "Manage Clusters",
+		Short: "Manage Kubernetes Clusters",
+		Long:  "Manage Kubernetes Clusters",
 		Annotations: map[string]string{
-			types.TagCommandType: types.TypeSystem,
```
Comment (Somefive) [inline references/cli/cluster.go:77]: Why Cluster Command belongs to CD?

**Review by Somefive — COMMENTED** [review]
`order` is added for sort commands?

File: references/cli/cluster.go (hunk 1 above)
Comment (wonderflow) [reply references/cli/cluster.go:77]: It belongs to the CD scenario for Deliver multi-cluster Application

**Comment by wonderflow** [comment]
> `order` is added for sort commands?

yes

**Review by Somefive — APPROVED** [review]

**Review by leejanee — APPROVED** [review]
LGTM

**Pull request merged**